contracts/vaults/BaseVaults/RibbonThetaVault.sol

```diff
@@ -368,7 +368,7 @@ contract RibbonThetaVault is RibbonVault, RibbonThetaVaultStorage {
      * @notice Sets the next option the vault will be shorting, and closes the existing short.
      *         This allows all the users to withdraw if the next option is malicious.
      */
-    function commitAndClose() external nonReentrant {
+    function commitAndClose() internal nonReentrant {
```

Comment:
Mostly for context: what motivated this change? Will we not need to call `commitAndClose()` externally anymore?

PS: `RibbonThetaVault.ts` test not passing with this change